recipe/build.bat

```diff
@@ -2,6 +2,7 @@ cd %SRC_DIR%/src
 call msvcbuild.bat amalg || goto :error
 copy luajit.exe %LIBRARY_BIN%
 copy lua51.dll %LIBRARY_BIN%
+mkdir -p %LIBRARY_LIB%
```

Comment:
The folder didnt exist which caused the lines below to copy the `.lib` file as if `lib` is a file. By first creating the directory the files are properly placed inside the directory.